Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM rust:1.54.0-buster as planner
+WORKDIR /app
+RUN curl https://getsubstrate.io -sSf | bash -s \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+COPY . .
+RUN cargo build --release \
+    && mv target/release/node-template /usr/local/bin/supra-dht \
+    && mv /usr/local/cargo/bin/sub* /usr/local/bin/ \
+    && rm -rf target \
+    && rm -rf /root/.cargo/registry/ \
+    && rm -rf /tmp && rm -rf /usr/local/rustup && rm -rf /usr/local/cargo
+
+FROM rust:1.54.0-buster as runtime
+COPY --from=planner /usr/local/bin/supra-dht /usr/local/bin/
+COPY --from=planner /usr/local/bin/sub* /usr/local/bin/
+EXPOSE 30333 9933 9944
+CMD ["supra-dht", \
+  "--base-path", "/tmp/alice", \
+  "--chain", "local", \
```

Comment:
Don’t pass all these hard coded params. Move them external and pass to docker image instead.